File: 3.Time Complexity/PermMissingElem.py
# 100/100
def solution(A):
    length = len(A)
    if length == 0:
    	return 1
    A.sort()
    for i in range(0, length -1):
    	if A[i] != i + 1:
    		return i + 1
    return length + 1
	

File: 3.Time Complexity/PermMissingElem.java
 import java.util.*;

// 100/100
class Solution {
    public int solution(int[] A) {
        if (A.length == 0) {
            return 1;
        }
        Arrays.sort(A);
        for (int i = 0; i < A.length; i++) {
            if (A[i] != i + 1) {
                return i + 1;
            }
        }
        return A.length + 1;
    }
}